flight/mcu/lib/teensyShared/thresholds.h

```diff
@@ -10,22 +10,54 @@
 // Transition from boost to coast if low accleration detected for 300 ms
 static const float coast_time_thresh = 300;
 
+// transition from coast to apogee if low velocity detected for 100ms
+static const float apogee_time_thresh = 100;
+
+static const float landing_time_thresh = 1000;
```

Comment:
Landing time thresh should be higher just in case. Detecting landing a second too late is better than detecting it too early.